pkg/graph/schema.graphql

```diff
@@ -1176,6 +1176,44 @@ type CurrentUser {
   launchDarkly: LaunchDarklySettings!
 }
 
+"""
+Represents a contact associated with a system intake
+"""
+type SystemIntakeContact {
+  euaUserId: String!
+  systemIntakeId: UUID!
+}
+
```

Comment:
For context, I mention that mutation being modified because the Figma mockups show that the page that this content will be added in is where that mutation is called. This should probably be an optional field on this mutation for now, since the UI for it hasn't been built yet.